dbt/models/spatial/docs.md

```diff
@@ -106,6 +106,39 @@ Cook County coordinated care area boundaries.
 **Geometry:** `MULTIPOLYGON`
 {% enddocs %}
 
+# corner
+
+{% docs table_corner %}
+CCAO corner lot indicator. Determined algorithmically by unobstructed access to
+perpendicular streets.
+
+The algorithm is:
+
+1. Find the minimum rectangle that bounds the parcel, then use that
+   rectangle to determine the parcel's orientation and length. These values
+   are used in the next step to draw a cross on the parcel.
+2. Draw a cross that cuts the parcel in four and extends out slightly
+   beyond the minimum rectangle of the parcel. We will look for parcels and
+   streets that intersect this cross in order to determine neighbors. Use the
+   bearing and length calculated in the previous step to draw the cross (i.e
+   the long side of the cross is parallel to the minimum bounding rectangle).
+3. Find the arms of each cross which intersect neighboring parcels
+   which _also_ touch the cross-originating PIN. The idea here is that cross
+   cross arms that intersect neighboring (touching) parcels are likely to be
```

Comment:
**[Nitpick, non-blocking]** Tiny dupe here:

```suggestion
   which _also_ touch the cross-originating PIN. The idea here is that
   cross arms that intersect neighboring (touching) parcels are likely to be
```

Reply:
Fixed in 6c7a4c4f!